templates/block_creator/schema.capnp

```diff
@@ -7,4 +7,5 @@ struct Schema {
     walletSeed @3: Text;
     walletPassphrase @4: Text;
     walletAddr @5: Text; # address of the wallet
+    network @6: Text="standard"; # network to join
```

Comment:
When we add a property to the schema with a default value, does it then get applied automatically to existing service schema's that are already running?

Reply:
no